source-analysis.md

```diff
@@ -1,36 +1,47 @@
-# Source analysis (Individual Assignment)
-
-Delete this introductory material when submitting your assignment
-Locate a digital source that is related to you (or your group's) general research aims. I will allow a generous leeway in terms of how much the source relates to your final topic, so even a source that is indirectly related is acceptable for now. Remember, although researches start off with a topic of interest, they nearly always end up changing their questions.
-
-This exercise is designed to (a) get you thinking critically about an digital source and (b) what are the possibilities and problems of using that source to illuminate your possible research interests. If in the process of doing your source analysis you realize that your research interest might need significant reworking, that is okay. I expect everyone’s topics to change over the course of the semester. The aim of this exercise is to teach you to think about how to approach *any* digital source.
-
-Overall, your answers should be *roughly* the equivalent of 2 pages, double-spaced.
-
-See the [README.md](README.md) file for information on how to submit this assignment.
+# Source analysis (Stephen DeCasien)
 ---
-
 ## Basic Source Information
 
 * Where is your source located (URL? Or traditional media)
+http://www.obelisks.org/en/index.htm
+
 * What kind of source is it? (Diaries, records of objects, data tables, etc...)
+The website is a summary of the information and data on each obelisk worldwide, based on this persons own travels. It has a list of obelisks that are linked to their present site with a multitude of information concerning each obelisk.
+
 * Who, or what organization, is responsible for hosting the source (if online), who gathered the information?
+This was a personal project of two "obelisks fans". There names are Hiroyuki Nagase and Shoji Okamoto. It seems that Hiroyuki Nagase did most of the work visiting each obelisk and collecting the data about them.
 
 ---
 
 ## Authorship
 
 * Is there any information available discussing the process by which the information was gathered, or how the digital project was constructed?
-* How transparent is the project about the choices they made in gathering and transforming the data for digitalization? [Click here for an example of a an excellent 'About' page from a database about Anglo-Saxons](http://www.pase.ac.uk/about/index.html)
+The author of the website states that he went to these obelisks to observe and take data on there present condition. It seems that some of the information has been taken from Wikipedia, Google Maps, and linked to different websites related to each individual obelisk.
```

Comment:
That is interesting and unusual for a database about ancient monuments! However, it does seem that the site is not very transparent in general. You might note that there is little justification for the manner of presentation (unlike an academic site).